kubernetes-ts-multicloud/README.md

```diff
@@ -0,0 +1,82 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Kubernetes Multi-cloud Example
+
+This example creates managed Kubernetes clusters using AKS, EKS, and GKE, and deploys an application
+on each cluster.
+
+## Deploying the App
+
+To deploy your infrastructure, follow the below steps.
+
+### Prerequisites
+
+1. [Install Pulumi](https://www.pulumi.com/docs/reference/install/)
+2. [Install Node.js 8.11.3](https://nodejs.org/en/download/)
+3. (Optional) [Configure AWS Credentials](https://www.pulumi.com/docs/reference/clouds/aws/setup/)
+4. (Optional) [Configure Azure Credentials](https://www.pulumi.com/docs/reference/clouds/azure/setup/)
+5. (Optional) [Configure GCP Credentials](https://www.pulumi.com/docs/reference/clouds/gcp/setup/)
+6. (Optional) [Configure local access to a Kubernetes cluster](https://kubernetes.io/docs/setup/)
+
+### Steps
+
+After cloning this repo, from this working directory, run these commands:
+
+1. Install the required Node.js packages:
+
+    ```bash
+    $ npm install
+    ```
+
+2. Create a new stack, which is an isolated deployment target for this example:
+
+    ```bash
+    $ pulumi stack init
+    ```
+
+3. Set the required configuration variables for this program:
+
+    ```bash
+    $ pulumi config set aws:region us-west-2
+    $ pulumi config set azure:location westus2
+    ```
+   
+   Note that you can choose different regions here.
+
+   We recommend using `us-west-2` to host your EKS cluster as other regions (notably `us-east-1`) may have capacity
+   issues that prevent EKS clusters from creating:
+
+    ```
+    Diagnostics:
+      aws:eks:Cluster: eksCluster
+        error: Plan apply failed: creating urn:pulumi:aws-ts-eks-example::aws-ts-eks::EKSCluster$aws:eks/cluster:Cluster::eksCluster: error creating EKS Cluster (eksCluster-233c968): UnsupportedAvailabilityZoneException: Cannot create cluster 'eksCluster-233c968' because us-east-1a, the targeted availability zone, does not currently have sufficient capacity to support the cluster. Retry and choose from these availability zones: us-east-1b, us-east-1c, us-east-1d
+            status code: 400, request id: 9f031e89-a0b0-11e8-96f8-534c1d26a353
+    ```
+
+    We are tracking enabling the creation of VPCs limited to specific AZs to unblock this in `us-east-1`: pulumi/pulumi-awsx#32
+
+4. (Optional) Choose which clusters to deploy, and uncomment the corresponding lines in the `index.ts` file. None of the
+   clusters are enabled by default.
+
+5. Bring up the stack, which will create the selected managed Kubernetes clusters, and deploy an application to each of
+   them.
+
+    ```bash
+    $ pulumi up
+    ```
+   
+   Here's what it should look like once it completes:
+   ![appUrls](images/appUrls.png)
```

Comment:
Why are these `appUrls` formatted the same with a `-`: e.g.: for the `aks` URL it is `url: http:/51.143.16.222.80/-/env"`

Is this something `kuard` does?

Reply:
Yes, that's how the URL is formed for the page I wanted in kuard.